wgsl/index.bs

```diff
@@ -10003,11 +10001,10 @@ between atomic accesses acting on different memory locations.
 
 Atomic built-in functions `must` not be used in a [=vertex=] shader stage.
 
-The storage class `SC` of the `atomic_ptr` parameter in all atomic built-in
-functions `must` be either [=storage classes/storage=] or [=storage
-classes/workgroup=].
-[=storage classes/workgroup=] atomics have a **Workgroup**
-[=memory scope=] in SPIR-V, while [=storage classes/storage=] atomics have a
+The address space `SC` of the `atomic_ptr` parameter in all atomic built-in
```

Comment:
I filed a TODO to update the "SC" pattern variables to "AS".

Reply:
I hesitated to call it "AS" because there is a different parameter (access mode) called "A".
I wanted to try just "S" (for "space"), so let me know if that doesn't work for you :)